tests/group/group_result_samples/add_group_result_error.json

```diff
@@ -4,14 +4,17 @@
       "name": "TESTGRPP0",
       "operation": "set",
       "requestId": "GroupRequest",
-      "error": {
-        "errorFunction": 10,
-        "errorCode": 2000,
-        "errorReason": 68,
-        "errorMessage": "Invalid attribute value specified.",
-        "errorOffset": 149,
-        "textInError": "name"
-      }
+      "commands": [
+        {
+          "safReturnCode": 10,
+          "returnCode": 2000,
+          "reasonCode": 68,
+          "image": "<securityrequest xmlns=\"http://www.ibm.com/systems/zos/saf\" xmlns:racf=\"http://www.ibm.com/systems/zos/racf\"><group name=\"TESTGRPP0\" operation=\"set\" requestid=\"GroupRequest\" /></securityrequest>",
```

Comment:
Why does "image" contain an xml string? If we don't have a valid value for "image" from the original response maybe we should just set this to `None`?

Reply:
I made the intentional choice to put the request_xml there for IRRSMO00 errors. This is because 1) there is no technical command image for these as no command image was generated, IRRSMO00 returned before we got that far and 2) these errors often reference specific offsets into the input xml string where there are concerns, so I wanted a way to tether the input xml to the error. I could be open to changing this to its own key in these error dictionaries.

Reply:
Alright, that makes sense. Change made (including changes to NullResponseError).

Reply:
These errors have been reverted back to their original format and are now included as part of `DownstreamFatalError` instead of `SecurityRequestError`.